internal/services/jsonrpc/server.go

```diff
@@ -182,6 +192,30 @@ func jsonRPCTimeoutHandler(h http.Handler, timeout time.Duration) http.Handler {
 	return http.TimeoutHandler(h, timeout, string(respMsg))
 }
 
+// corsHandler adds CORS headers to the response. We don't need sophisticated
+// cors handling here (not really kwild's concern, there should be other services
+// like LBs or KGW do that), so we just allow them.
+// NOTE: if this server is served behind KGW, those headers will be stripped.
+func corsHandler(h http.Handler) http.Handler {
+	allowMethods := "GET, POST, OPTIONS"
```

Comment:
Since we might support REST, I have 'GET' here. maybe we should add this only needed.

Reply:
This also remind me that we might want to return error if handlerV1 get a non-POST request?